src/components/Nav.js

```diff
@@ -17,9 +17,9 @@ function LeftSide({isLoggedIn, user, handleLogIn, handleLogOut}) {
   };
 
   return (
-    <FloatLeftMobileNav>
-      <Link style={{verticalAlign: "middle"}} to="/">
-        <img alt="open sauced" src={logo} />
+    <FloatLeftMobileNav >
```

Comment:
@JapneetSingh5 Remove space here too!

```suggestion
    <FloatLeftMobileNav>
```